variadic.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: default
+---
+
+# Variadic functions
+
+The example demonstrates variadic functions. Variadic functions are functions which accept a variable number of parameters.
+
+```
+package main
+
+import "fmt"
+
+/*
```

Comment:
Remove comments from here as it is already mentioned in description

Reply:
Should the single-line comments in `main` be removed too?

Reply:
No let them as it is